pkg/kubernetes/views/imagestream-body.html

```diff
@@ -1,3 +1,16 @@
+<div ng-repeat="statustags in stream.status.tags">
+  <div ng-repeat="condition in statustags.conditions" ng-if='condition.status == "False"'
+      class="alert alert-danger alert-dismissable">
```

Comment:
> Is this alert really dismissable?

No, `alert-dismissable` class appears to be broken. @dperpeet you alerted me to this in IRC. I'm not sure how to debug. Any more details on this?